components/Template.js

```diff
@@ -69,4 +67,44 @@ function transformData(fn, templateKey, originalData) {
   throw new Error('`transformData` must be a function or an object');
 }
 
+function renderTemplate({template, compileOptions, helpers, data}) {
+  var hogan = require('hogan.js');
+  var forOwn = require('lodash/object/forOwn');
+  var content;
+
+  if (typeof template !== 'string' && typeof template !== 'function') {
+    throw new Error('Template must be `string` or `function`');
+  }
+
+  if (typeof template === 'function') {
+    content = template(data);
+  }
+
+  if (typeof template === 'string') {
+    data = addTemplateHelpersToData(data);
+
+    content = hogan.compile(template, compileOptions).render(data);
+  }
+
+  // We add all our template helper methods to the template as lambdas. Note
+  // that lambdas in Mustache are supposed to accept a second argument of
+  // `render` to get the rendered value, not the literal `{{value}}`. But
+  // this is currently broken (see
+  // https://github.com/twitter/hogan.js/issues/222).
+  function addTemplateHelpersToData(templateData) {
```

Comment:
Ah I should have said that, I did not modify this function, we can refactor it later on but I did not touch it. I only ported it from utils/ to Template, since it was the only place using it.